README.md

```diff
@@ -63,10 +135,13 @@ Our [CONTRIBUTING](https://github.com/OpenLiberty/open-liberty/blob/master/CONTR
     
     ```open-liberty\dev\cnf\release\dev\openliberty\<version>\openliberty-xxx.zip```
 
-5. Go [Open issues](https://github.com/OpenLiberty/open-liberty/issues), [Review existing contributions](https://github.com/OpenLiberty/open-liberty/pulls), or [Submit fixes](https://github.com/OpenLiberty/open-liberty/blob/master/CONTRIBUTING.md).
+## License
+[![License](https://img.shields.io/badge/License-EPL%201.0-green.svg)](https://opensource.org/licenses/EPL-1.0)
+
+Usage is provided under the [EPL 1.0 license](https://opensource.org/licenses/EPL-1.0) See LICENSE for the full details.
 
-## Community
-1. [Open Liberty group.io](https://groups.io/g/openliberty)
-2. [OpenLibertyIO on Twitter](https://twitter.com/OpenLibertyIO)
-3. [open-liberty tag on stackoverflow](https://stackoverflow.com/questions/tagged/open-liberty)
 
+## Contact
```

Comment:
I don't think this section is needed. The group.io and stackoverflow links are available by clicking the badges up top, and the issue link is more prominently visible with the regular "Issues" tab that github shows at the top of the page